src/game_battlealgorithm.cpp

```diff
@@ -1149,8 +1149,9 @@ std::string Game_BattleAlgorithm::Skill::GetFailureMessage() const {
 }
 
 bool Game_BattleAlgorithm::Skill::IsReflected(const Game_Battler& target) const {
-	// Skills invoked by items ignore reflect
-	if (item) {
+	// Skills invoked by items and skills with the attribute
+        // "easyrpg_ignore_reflect" set ignore reflect
```

Comment:
> Add "easyrpg_ignore_reflect" attribute for skills 

Wrong spacing. Imo that entire comment is useless because it simply states what you can already see in the if. So delete it, thx.

Reply:
Fixed.